ui/app/rif/components/index.js

```diff
@@ -1,13 +1,32 @@
 import CustomButton from './customButton'
 import SearchDomains from './searchDomains'
 import AddNewTokenNetworkAddress from './modal/addNewTokenNetworkAddress/addNewTokenNetworkAddress'
-import { DomainIcon, LuminoNodeIcon, RifStorageIcon } from './commons'
+import {DomainIcon, LuminoNodeIcon, RifStorageIcon} from './commons'
+import Menu from './menu/index'
+import h from 'react-hyperscript'
+import CustomModal from './modal/modal';
+
+function buildModal (modalComponent, currentModal) {
+  return h(modalComponent, {key: currentModal.name, message: currentModal.message})
```

Comment:
Do we really need to use hyperscript in our components?

Reply:
In this case yes because the parent component does that, it's just to keep the structure of the parent app component that's all.